contracts/ProviderPool.sol

```diff
@@ -4,8 +4,22 @@ import "./SortedDoublyLL.sol";
 import "zeppelin-solidity/contracts/ownership/Ownable.sol";
 
 contract ProviderPool is Ownable {
+
+  // TODO: init
```

Comment:
is it safe to initialize to 0 explicitly? what do you mean by init

Reply:
I meant set a default value in the contructor of the main contract